packages/react-dom/src/__tests__/ReactFunctionComponent-test.js

```diff
@@ -158,16 +158,12 @@ describe('ReactFunctionComponent', () => {
     expect(function() {
       ReactTestUtils.renderIntoDocument(<Child test="test" />);
     }).toThrowError(
-      __DEV__
-        ? 'Function components cannot have string refs. We recommend using useRef() instead.'
-        : // It happens because we don't save _owner in production for
-          // function components.
-          'Element ref was specified as a string (me) but no owner was set. This could happen for one of' +
-            ' the following reasons:\n' +
-            '1. You may be adding a ref to a function component\n' +
-            "2. You may be adding a ref to a component that was not created inside a component's render method\n" +
-            '3. You have multiple copies of React loaded\n' +
-            'See https://fb.me/react-refs-must-have-owner for more information.',
+      'Element ref was specified as a string (me) but no owner was set. This could happen for one of' +
```

Comment:
This message was previously inconsistent between DEV and PROD. Now we show the same message.

Reply:
Revert